.github/workflows/e2e-tests.yml

```diff
@@ -149,102 +125,47 @@ jobs:
-          kubectl rollout status deployment/oncall-ci-grafana --timeout=300s
-          kubectl rollout status deployment/oncall-ci-engine --timeout=300s
-          kubectl rollout status deployment/oncall-ci-celery --timeout=300s
+          make cluster/up
 
-      - name: Run e2e Tests
-        env:
-          # BASE_URL represents what is accessed via a browser
-          BASE_URL: http://localhost:30002/grafana
-          # ONCALL_API_URL is what is configured in the plugin configuration form
-          # it is what the grafana container uses to communicate with the OnCall backend
-          #
-          # 172.17.0.1 is the docker bridge network default gateway. Requests originate in the grafana container
-          # hit 172.17.0.1 which proxies the request onto the host where port 30001 is the node port that is mapped
-          # to the OnCall API
-          ONCALL_API_URL: http://172.17.0.1:30001
-          GRAFANA_ADMIN_USERNAME: oncall
-          GRAFANA_ADMIN_PASSWORD: oncall
-          GRAFANA_EDITOR_USERNAME: editor
-          GRAFANA_EDITOR_PASSWORD: editor
-          GRAFANA_VIEWER_USERNAME: viewer
-          GRAFANA_VIEWER_PASSWORD: viewer
-          MAILSLURP_API_KEY: ${{ secrets.MAILSLURP_API_KEY }}
-          BROWSERS: ${{ inputs.browsers }}
-        working-directory: ./grafana-plugin
-        run: yarn test:e2e
-
-      - name: Run expensive e2e Tests
-        if: inputs.run-expensive-tests
-        env:
-          BASE_URL: http://localhost:30002/grafana
-          ONCALL_API_URL: http://172.17.0.1:30001
-          GRAFANA_ADMIN_USERNAME: oncall
-          GRAFANA_ADMIN_PASSWORD: oncall
-          GRAFANA_EDITOR_USERNAME: editor
-          GRAFANA_EDITOR_PASSWORD: editor
-          GRAFANA_VIEWER_USERNAME: viewer
-          GRAFANA_VIEWER_PASSWORD: viewer
-          MAILSLURP_API_KEY: ${{ secrets.MAILSLURP_API_KEY }}
-        working-directory: ./grafana-plugin
-        run: yarn test:e2e-expensive
-
-      # spit out the engine, celery, and grafana logs, if the the e2e tests have failed (or were flaky)
-      # can be helpful for debugging these tests
-      # GitHub Action reference: https://github.com/jupyterhub/action-k8s-namespace-report
-      - name: oncall-engine logs
-        if: always()
-        uses: jupyterhub/action-k8s-namespace-report@v1
-        with:
-          important-workloads: deploy/oncall-ci-engine
+      - name: Install Playwright deps
+        uses: docker://mcr.microsoft.com/playwright:next-jammy
 
-      - name: oncall-celery logs
-        if: always()
-        uses: jupyterhub/action-k8s-namespace-report@v1
-        with:
-          important-workloads: deploy/oncall-ci-celery
-
-      - name: grafana logs
-        if: always()
-        uses: jupyterhub/action-k8s-namespace-report@v1
-        with:
-          important-workloads: deploy/oncall-ci-grafana
+      - name: Tilt CI
+        shell: bash
+        env:
+          GRAFANA_IMAGE_TAG: ${{ inputs.grafana-image-tag }}
+        run: tilt ci
```

Comment:
we'd likely need a way here to conditionally run the "expensive" tests

Reply:
Will do it, thank you